### DISCUSSION TITLE
Apache Cloud Stack Bug Report - KeyStore KVM Hosts ( Urgent )

### DISCUSSION THREAD

**Discussion by ghost** [apache/cloudstack#9018] [Q&A]
# Issue Type 
```
Bug Report 
```
# COMPONENT NAME
```
ADD HOST
```
# CLOUDSTACK VERSION
```
4.19.0.1
```
# CONFIGURATION
```
Basic Network 
```
# OS / ENVIRONMENT
```
Ubuntu 22.04 
```

>Issue 
>`KeyStore KVM Host : ( My Ip )`

~~Request ~~

**Apache Cloud Stack has some inconveniences such as KeyStore KVM Host Issue
How can we do that again 
Can you know why it happened**

# The issues we have are resolved below 

![Screen Shot 2024-04-23 at 3.35.21 PM](https://hackmd.io/_uploads/BJY15lH-A.png)

*We have that problem, what kind of solution we can get for that, what kind of solution can we solve it *

*I would like to know what happened to the butt Apache Cloud Stack because it is not convenient to post it *

## Please tell me what you should do about this bug

# Thank You

**Comment by yadvr** [comment]
@therev3l4 did you allow root ssh based login with password or ssh-key? What installation guide or steps have you followed ?

**Comment by yadvr** [comment]
@therev3l4 the general steps to debug this is to look at the management-server and agent logs to see why it failed. Since this looks like an initial setup issue with no details on reproducing the issue, I'm moving as discussions wherein users ML can also participate in this.
